speech/google/cloud/speech/client.py

```diff
@@ -195,7 +196,9 @@ def sync_recognize(self, sample, language_code=None,
             method='POST', path='speech:syncrecognize', data=data)
 
         if len(api_response['results']) == 1:
-            return api_response['results'][0]['alternatives']
+            return [Transcript.from_api_repr(alternative)
+                    for alternative
+                    in api_response['results'][0]['alternatives']]
```

Comment:
While you're here, can you make a separate variable

```python
result = api_response['results'][0]
```

which "confirms" the `len(api_response['results']) == 1` check and then in your list comprehension here you'd just say `result['alternatives']`.